packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -34,12 +34,12 @@ export function useFrequentlyBoughtTogether<TObject>({
   const objectIDs = useStableValue(userObjectIDs);
   const queryParameters = useStableValue(userQueryParameters);
 
-  const initialState = useStableValue<GetRecommendationsResult<TObject>>({
+  const initialResults = useStableValue<GetRecommendationsResult<TObject>>({
     recommendations: [],
     ...userInitialResults,
   });
   const [result, setResult] = useState<GetRecommendationsResult<TObject>>(
-    initialState
+    initialResults
```

Comment:
There's still a vocabulary mismatch. Here we're storing results (plural) in result.

Should we go with the singular form?

Reply:
ah, it is a bit weird 🤔 
something like we have `initialReuslts` cuz they represent the intial recommendation**s** provided (as of now) by the user and we have `result` as in the API result .... wdyt?

Reply:
initialResult everywhere would make sense to me, it's one result object which has multiple recommendations inside

Reply:
I agree with @Haroenv.

Reply:
✅ : https://github.com/algolia/recommend/pull/109/commits/adffbda5888120c89018516ff360980a7fc493a1